src/bun.js/api/bun/subprocess.zig

```diff
@@ -1406,13 +1408,20 @@ pub const Subprocess = struct {
                         @panic("This shouldn't happen");
                     }
 
-                    // process has already exited
-                    // https://cs.github.com/libuv/libuv/blob/b00d1bd225b602570baee82a6152eaa823a84fa6/src/unix/process.c#L1007
-                    subprocess.onExitNotification();
+                    send_exit_notification = true;
```

Comment:
@dylan-conway i'm wondering and hoping this partially addresses the reliability issues with Bun.spawn